### ISSUE TITLE
doesn't detect the same id using error and not showing the red dot in any of the errors

### ISSUE THREAD

**Issue by Salman1319** [AtomLinter/linter-eslint#1431]
## Bug Checklist
- [ ] Restart Atom
- [ ] Verify the `eslint` CLI gives the proper result, while `linter-eslint` does not
- [ ] Paste the output of the `Linter Eslint: Debug` command from the Command Palette below

```
Linter Eslint: Debug output here
```

**Comment by UziTech** [comment]
The issue template helps us identify the error. If you would fill it out we can reopen this.
